notebooks-auth/geoserver.ipynb

```diff
@@ -530,24 +542,24 @@
      "output_type": "stream",
      "text": [
       "Detail:\n",
-      "  Resource: [https://pavics.ouranos.ca/geoserver-secured/test_workspace/wfs?typeNames=test_workspace:Espace_Vert&request=GetFeature]\n",
-      "  User:     [test-user-e0ad80ca-61aa-44f3-807a-e6d95a6d9b8d]\n",
-      "  Code:     [401]\n",
+      "  Resource: [https://pavics.ouranos.ca/geoserver/test_workspace/wfs?typeNames=test_workspace:Espace_Vert&request=GetFeature]\n",
+      "  User:     [test-user-58d91ff0-48d8-42fe-a121-29302959b295]\n",
+      "  Code:     [403]\n",
```

Comment:
Normal that the http response code change from 401 to 403?

Reply:
Yes, I think it's normal since the requests now go through Twitcher. It seems to send a 403 response if the request access is denied : https://github.com/bird-house/twitcher/blob/aca67f1fb7f9014ccb8718666b865287da7c61d5/twitcher/owsproxy.py#L189